src/base/bittorrent/private/speedmonitor.cpp

```diff
@@ -27,16 +27,24 @@
  * exception statement from your version.
  */
 
-#include <QList>
 #include "speedmonitor.h"
 
 void SpeedMonitor::addSample(const SpeedSample &sample)
 {
-    m_speedSamples.push_back(sample);
+    static_assert(MAX_SAMPLES > 0, "MAX_SAMPLES should > 0");
```

Comment:
Guys, think again! Go get some sleep - let the mind come back to you. 😊 Why is it here? If you don't trust yourself, comment near constant will be enough.